tests/LinAlg/vectorTest.hpp

```diff
@@ -85,11 +92,11 @@ namespace hiop { namespace tests {
  * and return `false` otherwise.
  *
  */
-class VectorTests : public TestBase
+class VectorTest : public TestBase
 {
 public:
-  VectorTests(){}
-  virtual ~VectorTests(){}
+  VectorTest(){}
```

Comment:
What's the rationale for removing the plural here?

Reply:
This was done in debugging, and I am happy to make it plural again if desired.